application/common/components/passwordStore/Ldap.php

```diff
@@ -413,19 +413,17 @@ public function getSearchCriteria()
 
     /**
      * @param string $employeeId
-     * @return LDAP_Entry|array
+     * @return LDAP_Entry
      * @throws \common\components\passwordStore\UserNotFoundException
      */
-    public function findUser(string $employeeId): LDAP_Entry|array
+    public function findUser(string $employeeId): LDAP_Entry
     {
         $this->connect();
         $criteria = $this->getSearchCriteria();
-        $client = $this->ldapClient;
 
         try {
             /** @var LDAP_Entry $user */
-            $user = $client->query()
-                ->select($criteria)
+            $user = LDAP_Entry::select($criteria)
```

Comment:
It looks like this no longer uses the `ldapClient` that is connected to the applicable LDAP server. Does LDAP_Entry::select() have some way to get access to the correct LDAP server / connection details? I'm a little confused here.